2022-04-28 11:56:38,557 INFO    MainThread:25404 [wandb_setup.py:_flush():75] Loading settings from C:\Users\gizem\.config\wandb\settings
2022-04-28 11:56:38,558 INFO    MainThread:25404 [wandb_setup.py:_flush():75] Loading settings from C:\Users\gizem\Documents\GitHub\noisy_fair\wandb\settings
2022-04-28 11:56:38,559 INFO    MainThread:25404 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-04-28 11:56:38,560 INFO    MainThread:25404 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'sparse_gt_variance.py', 'program': 'c:\\Users\\gizem\\Documents\\GitHub\\noisy_fair\\sparse_gt_variance.py'}
2022-04-28 11:56:38,562 INFO    MainThread:25404 [wandb_init.py:_log_setup():438] Logging user logs to C:\Users\gizem\Documents\GitHub\noisy_fair\wandb\run-20220428_115638-29oxcun7\logs\debug.log
2022-04-28 11:56:38,564 INFO    MainThread:25404 [wandb_init.py:_log_setup():439] Logging internal logs to C:\Users\gizem\Documents\GitHub\noisy_fair\wandb\run-20220428_115638-29oxcun7\logs\debug-internal.log
2022-04-28 11:56:38,568 INFO    MainThread:25404 [wandb_init.py:init():472] calling init triggers
2022-04-28 11:56:38,569 INFO    MainThread:25404 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {'n_train': 100, 'n_test': 10000.0, 'n_features': 31, 'tau': 1.0, 'SNR': 10, 'n_informative': 31, 'n_redundant': 0, 'n_repeated': 0, 'n_useless': 0, 'n_clusters_per_class': 1, 'class_sep': 3.1622776601683795, 's': 1, 'label_noise_prob': 0}
2022-04-28 11:56:38,572 INFO    MainThread:25404 [wandb_init.py:init():525] starting backend
2022-04-28 11:56:38,572 INFO    MainThread:25404 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-04-28 11:56:38,631 INFO    MainThread:25404 [backend.py:ensure_launched():219] starting backend process...
2022-04-28 11:56:39,102 INFO    MainThread:25404 [backend.py:ensure_launched():224] started backend process with pid: 27348
2022-04-28 11:56:39,105 INFO    MainThread:25404 [wandb_init.py:init():534] backend started and connected
2022-04-28 11:56:39,148 INFO    MainThread:25404 [wandb_init.py:init():598] updated telemetry
2022-04-28 11:56:39,267 INFO    MainThread:25404 [wandb_init.py:init():629] communicating run to backend with 30 second timeout
2022-04-28 11:57:09,284 INFO    MainThread:25404 [interface_shared.py:_communicate_run():325] couldn't get run from backend
2022-04-28 11:57:09,285 ERROR   MainThread:25404 [wandb_init.py:init():634] backend process timed out
2022-04-28 11:57:09,286 ERROR   MainThread:25404 [wandb_init.py:init():645] encountered error: Error communicating with wandb process
try: wandb.init(settings=wandb.Settings(start_method='fork'))
or:  wandb.init(settings=wandb.Settings(start_method='thread'))
For more info see: https://docs.wandb.ai/library/init#init-start-error
2022-04-28 11:59:07,909 INFO    MainThread:25404 [wandb_init.py:teardown():286] tearing down wandb.init
